src/instrument.js

```diff
@@ -107,8 +126,38 @@ async function findInstruments(oreAccountName, activeOnly = true, category = und
   return instruments;
 }
 
+function sortInstruments(instruments, rightName, sortOrder = "cheapestThenMostRecent") {
+  // sorts the instruments depending on the search criteria : 
+  // cheapestThenMostRecent - returns the cheapest instrument for the right and if there are more than one with the same price, then returns the latest created/updated instrument
+  // mostRecent - returns the latest instrument created/updated for the right
+  let sortedInstruments;
+  let cheapestInstrument;
+  let cheapestPrice;
+  let cheapestInstruments;
+
+  const sortTypes = {
```

Comment:
We don't need this. Just check for the string value below. Would be nice to have enums but not worth it until we move to Typescript.